scripts/BUILD.bazel

```diff
@@ -46,3 +46,34 @@ kt_jvm_test(
         "//third_party:org_jetbrains_kotlin_kotlin-test-junit",
     ],
 )
+
+kt_jvm_binary(
+    name = "xml_syntax_check",
+    testonly = True,
+    main_class = "org.oppia.android.scripts.XMLSyntaxCheck$Companion",
+    runtime_deps = [":xml_syntax_check_lib"],
+)
+
+kt_jvm_library(
+    name = "xml_syntax_check_lib",
+    testonly = True,
+    srcs = [
+        "src/java/org/oppia/android/scripts/RepoFile.kt",
```

Comment:
This should be in its own library & added as a dep so it can be shared between different scripts that need it. Ditto for any of the other generic classes (prefer using libraries over duplicating sources--this will make the build perform more efficiently).

Reply:
Outdated.